dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops

```diff
@@ -200,12 +200,22 @@ UPDATE_WRONG_PROCESS_TYPE_DEPENDENCY_CLIENT_ERROR=CWWKF0041E: The {0} feature on
 UPDATE_WRONG_PROCESS_TYPE_DEPENDENCY_CLIENT_ERROR.explanation=The configured feature does not support the specified process type.
 UPDATE_WRONG_PROCESS_TYPE_DEPENDENCY_CLIENT_ERROR.useraction=Remove the feature from the configuration or change the feature manifest file to enable support for the desired process type.
 
-# CWWK042E  {0} = feature Name: {1} = server name 
-UPDATE_MISSING_CORE_FEATURE_ERROR=CWWKF0042E: A feature definition cannot  be found for the  {0} feature.  Try running the command, bin/installUtility install {0},  to install the feature. Alternatively, you can run the command, bin/installUtility install {1},  to install all features that are referenced by this configuration.
+# CWWK0042E  {0} = feature Name: {1} = server name 
+UPDATE_MISSING_CORE_FEATURE_ERROR=CWWKF0042E: A feature definition cannot be found for the {0} feature.  Try running the command, bin/installUtility install {0}, to install the feature. Alternatively, you can run the command, bin/installUtility install {1}, to install all features that are referenced by this configuration.
 UPDATE_MISSING_CORE_FEATURE_ERROR.explanation=The feature manager could not find a definition for the specified feature.
 UPDATE_MISSING_CORE_FEATURE_ERROR.useraction=Check that the name of the specified feature is spelled correctly.  \
-Also verify that the feature definition exists and is readable in the lib/features directory. If the feature does not exist,  try running the command, bin/installUtility install feature_name,  to install the feature \
-or try running the command, bin/installUtility install server_name, to install all features that are referenced by this configuration. 
+Also verify that the feature definition exists and is readable in the lib/features directory.  If the feature does not exist, try running the command, bin/installUtility install feature_name, to install the feature \
+or try running the command, bin/installUtility install server_name, to install all features that are referenced by this configuration.
+
+# CWWK0043E  {0} = EePlatform1: {1} = Eeplatform2: {2} = configuredFeature1: {3} = configuredFeature2: {4} shared EePlaform 
+UPDATE_CONFLICT_INCOMPATIBLE_EE_FEATURES_SAME_PLATFORM_ERROR=CWWKF0043E: The configured feature {2} of {0} is incompatible with the configured feature {3} of {1}.  The configuration includes an incompatible combination of {4} features and is not supported.  Update server.xml to remove the incompatible features.
+UPDATE_CONFLICT_INCOMPATIBLE_EE_FEATURES_SAME_PLATFORM_ERROR.explanation=The server configuration contains an incompatible combination of features that support the Jakarta EE or Java EE programming models. This conflict can happen if the two incompatible features are configured directly in the server configuration. Or, if the configured features include different versions of a feature that are incompatible.
```

Comment:
change
in the server configuration. Or, if the configured features include different versions of a feature that are incompatible.
to
in the server configuration, or if the configured features include different versions of a feature that are incompatible.

Reply:
Done.